crates/python/src/qpu/client.rs

```diff
@@ -61,8 +197,11 @@ impl PyQcsClient {
                 Some(use_gateway) => client.with_use_gateway(use_gateway),
             };
 
-            Python::with_gil(|py| <_ as ToPython<Py<PyAny>>>::to_python(&Self(client), py))
-        })?
-        .extract()
+            Ok(Self(client))
+        })
+    }
+
+    fn info(&self) -> String {
```

Comment:
There is an `impl_repr!` macro that will implement this as a `__repr__` method for you.

Is this method replacing some functionality in its Python counterpart?

Reply:
I'll use that - overall I'm hesitant to expose what may seem at first to be a serializable format but is actually just rust `Debug` output on the off chance that someone needs to parse it. I don't think serialization should be a common or recommended usecase, but I don't want support for it to be ambiguous 

```
Qcs {
  config: ClientConfiguration {
    tokens: Mutex {
      data: Tokens {
        bearer_access_token: Some("..."),
        refresh_token: Some("...")
      }
    },
    api_url: "https://api.qcs.rigetti.com", 
    auth_server: AuthServer {
      client_id: "0oa3ykoirzDKpkfzk357",
      issuer: "https://auth.qcs.rigetti.com/oauth2/aus8jcovzG0gW2TUG355"
    },
    grpc_api_url: "https://legacy.grpc.qcs.rigetti.com",
    quilc_url: "tcp://127.0.0.1:5555",
    qvm_url: "http://127.0.0.1:5000"
  },
  use_gateway: true
}
```

Reply:
Yeah, definitely a fair concern. If we switch to `impl_repr!` we can at least keep this PR moving, and consider it as a potential improvement to `rigetti-pyo3` that will in turn benefit all of these bindings.

Reply:
I liked your idea from https://github.com/rigetti/qcs-sdk-rust/pull/235#discussion_r1083030237 👍